internal/uvm/create.go

```diff
@@ -145,11 +145,11 @@ func verifyOptions(ctx context.Context, options interface{}) error {
 				return errors.New("PreferredRootFSTypeVHD requires at least one VPMem device")
 			}
 		}
-		if opts.KernelDirect && osversion.Get().Build < 18286 {
+		if opts.KernelDirect && osversion.Build() < 18286 {
```

Comment:
Question: is there much overhead in the syscall that's made? (wondering if there would be benefits in setting a package-level variable to store the version in, or if that would not give (significant) benefits) 

https://github.com/golang/sys/blob/700132347e0702a3e5f100fba2752993b8de9600/windows/zsyscall_windows.go#L2266-L2273

```go
func GetVersion() (ver uint32, err error) {
	r0, _, e1 := syscall.Syscall(procGetVersion.Addr(), 0, 0, 0, 0)
	ver = uint32(r0)
	if ver == 0 {
		err = errnoErr(e1)
	}
	return
}
```

Reply:
I'll have to trial the overhead but one syscall will definitely be better than many 😄. You'd think we could just do this in an init, assign it to an exported global and be done with it.. Not sure if I'm missing something/some history on why we just call it over and over.

Reply:
@kevpar Should we move test/functional/manifest top level (or just out of test really), call `Build()` once in the osversion package and cache the result in an exported var and just use this everywhere? I don't have the history on why the manifest lives in test at the moment.

Reply:
The alternative could be to add a `sync.Once` in `osversion.Get()`; that way we don't have to add some "magic variable", and any consumer of that utility won't have to worry about it, and get the improvement "out of the box".

Reply:
That's what I have at the moment after firing up a new branch, I like it better

Reply:
I don't feel like re-rechanging what's used from osversion so this is nice

Reply:
Actually went ahead and added that as an extra commit in this branch (can remove if it's not needed)

Reply:
You're too quick for me 😎. Let's see what others think, I think it should be fine

Reply:
Switching to a global and `sync.Once` looks fine to me. I might be missing why moving `manifest` came up, though?

Reply:
No functional reason other than it feels weird to import this thing from the test dir when needed

Reply:
Ah, I thought it was related to this somehow. I'm in favor of moving it, but as a separate PR.